casts/texts/src/UIExpandableText/UIExpandableText.tsx

```diff
@@ -5,26 +5,29 @@ import { TouchableOpacity } from '@tonlabs/uikit.controls';
 import { uiLocalized } from '@tonlabs/localization';
 import type { UIExpandableTextProps } from './types';
 import { UILayout } from '../constants';
+import { MeasureLabel } from './MeasureLabel';
 
 function UIExpandableTextImpl(props: UIExpandableTextProps) {
     const { numberOfLines, testID, ...rest } = props;
     const theme = useTheme();
     const styles = useStyles(theme);
 
     const [isExpanded, setIsExpanded] = React.useState<boolean>(false);
+    const [isFit, setIsFit] = React.useState<boolean>(false);
 
     const onExpand = React.useCallback(() => {
         setIsExpanded(true);
     }, []);
 
     return (
         <View style={styles.container} testID={testID}>
+            <MeasureLabel {...props} setIsFit={setIsFit} />
```

Comment:
I wonder why you used separate `Text` component to render it, is it because of `numberOfLines`? 🤔

Reply:
Yes, in order to render the full text, measure its dimensions and find out the number of lines. I didn't find any other way to find out the number of rows on iOS and web. If you know any - please share.

Reply:
I might know, but not sure that it's easier :)

1. I wonder if it's possible to look at 3 dots at the end, as an indicator that text was truncated
2. There're some native APIs, it sth that I dream of for a while, but it definitely not an easy thing to do (and yep, it lacks web support)

I think your solution is ok, just please leave a comment about `numberOfLines` to explain why you decided to do it this way